src/compile/selection/selection.ts

```diff
@@ -239,17 +239,6 @@ export function channelSignalName(selCmpt: SelectionComponent, channel: Channel)
   return selCmpt.name + '_' + channel;
 }
 
-function clippedGroup(model: Model, marks: any[]): any[] {
-  return [{
-    type: 'group',
-    encode: {
-      enter: {
-        width: model.getSizeSignalRef('width'),
-        height: model.getSizeSignalRef('height'),
-        fill: {value: 'transparent'},
-        clip: {value: true}
-      }
-    },
-    marks: marks.map(model.correctDataNames)
-  }];
+function clipMarks(marks: any[]): any[] {
+  return marks.map((m) => (m.clip = true, m));
```

Comment:
Oh interesting that we don't need to clip anymore. 
(I wonder why?) 

Also, do we I need to add `fill: {value: 'transparent'}` to the `nested-main-group`